### ISSUE TITLE
Set page sizes in a combined pdf

### ISSUE THREAD

**Issue by vegarringdal** [galkahana/HummusJS#155]
Hi, thanks for making this npm module.

Is it possible to set page sizes in a combined pdf

Like set pages 1-4 to A4 and 5-10 to A3
Pages I try to convert will be between A5-A1.
Want to make something that makes it simpler to print combined pdf and let printer use paper size from pdf size

**Comment by galkahana** [comment]
Yes. You can modify page sizes. Look for examples in tests and issues

**Comment by vegarringdal** [comment]
great, will markup comments etc stay in correct size when you resize.

**Comment by galkahana** [comment]
how about you look and see. if something is wrong, you can always change it. changing the page size attribute will normally only change the bounds, there's no auto scaling for artifacts.

**Comment by eouw0o83hf** [comment]
Just so it shows up for anyone who searches: the necessary method is to set the `box` sizes on the created page. For example:

    const pageInfo = // pageInfo from a reference page
    const newPage = writer.createPage();

    newPage.mediaBox = pageInfo.getMediaBox();
    newPage.cropBox = pageInfo.getCropBox();
    newPage.trimBox = pageInfo.getTrimBox();
    newPage.bleedBox = pageInfo.getBleedBox();
    newPage.artBox = pageInfo.getArtBox();

Then you can add `newPage` into the document and go from there.
